basics.js

```diff
@@ -31,6 +37,16 @@ def greet(name, excited)
 end
 */
 
+var greet = function greeting (name, excited) {
+ var agreeting = "Hi, my name is #{name}";
```

Comment:
ES5/JavaScript doesn't have string interpolation, but I see what you're thinking.